Solutions/Azure Active Directory/Analytic Rules/FirstAppOrServicePrincipalCredential.yaml

```diff
@@ -24,7 +24,7 @@ tags:
   - NOBELIUM
 query: |
   AuditLogs
-  | where OperationName has_any ("Add service principal", "Certificates and secrets management") // captures "Add service principal", "Add service principal credentials", and "Update application - Certificates and secrets management" events
+  | where OperationName has ("Certificates and secrets management")
```

Comment:
The author of the query has seen instances of SP getting used for thus activity. If this brings in false positives in your environment, making changes in active analytical rule would be a better choice.